run-tests.sh

```diff
@@ -24,6 +24,6 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 pydocstyle invenio_workflows && \
-sphinx-build -qnNW docs docs/_build/html && \
+sphinx-build -qnN docs docs/_build/html && \
 python setup.py test && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+sphinx-build -qnN -b doctest docs docs/_build/doctest
```

Comment:
By looking at the error (https://travis-ci.org/inveniosoftware-contrib/invenio-workflows/jobs/227859081#L677) I am not sure which is the code of the warning we should disable.